overlord/configstate/configcore/netplan.go

```diff
@@ -92,18 +117,137 @@ func getNetplanFromSystem(key string) (result interface{}, err error) {
 	netplan := conn.Object("io.netplan.Netplan", "/io/netplan/Netplan")
 
 	if err := netplan.Call("io.netplan.Netplan.Config", 0).Store(&netplanConfigSnapshotBusAddr); err != nil {
-		if derr, ok := err.(dbus.Error); ok {
-			// Having no netplan config is *not* an error, we just
-			// do not support netplan config.
-			if isNoServiceOrMethodErr(derr) {
-				return nil, nil
-			}
+		return nil, err
+	}
+	logger.Debugf("using netplan config %v", netplanConfigSnapshotBusAddr)
```

Comment:
can this dump contain passwords? (wifi, vpn)

Reply:
This is "just" the dbus config bus path, I change the debug command to make this clearer.

Reply:
Yeah, silly me, I was confused by the name, not it's fine no need to change anything